scientific-computing-with-python/learn-lambda-functions-by-building-a-expense-tracker/expense-tracker.py

```diff
@@ -0,0 +1,54 @@
+expenses = []
+
+def add_expense(amount, category):
+    expenses.append({"amount": amount, "category": category})
+
+def list_expenses():
```

Comment:
I'd change `list_expenses` to `print_expenses` and make it accept one parameter with expenses to print. That way it can be used for all printing list of expenses needs in the project, instead of having printing in multiple places.

```suggestion
def print_expenses(expenses):
```

Reply:
I just tested all of this, and this is a great suggestion, thanks.